zuul.d/molecule.yaml

```diff
@@ -822,6 +822,15 @@
     - ^.config/molecule/.*
     name: cifmw-molecule-ci_lvms_storage
     parent: cifmw-molecule-noop
+- job:
+    files:
+    - ^common-requirements.txt
+    - ^test-requirements.txt
+    - ^roles/cifmw_external_dns/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
+    - ^.config/molecule/.*
+    name: cifmw-molecule-cifmw_external_dns
+    parent: cifmw-molecule-noop
```

Comment:
this was generated by make role_molecule requested in https://github.com/openstack-k8s-operators/ci-framework/pull/2053#issuecomment-2225177140